### ISSUE TITLE
Plugin stopped working on Yosemite

### ISSUE THREAD

**Issue by iamso** [whomwah/qlstephen#42]
Hi

I installed the plugin a while ago and everything was working fine, but now all of a sudden it isn't.
I didn't really change anything, at least not that I know of. And there was no big OS update or anything.
There also isn't any other QL plugin interfering, because if I remove all the other plugins and only keep QLStephen, it's still not working.

Is there anything you can think of, that could cause this issue? Or is there any way I can debug/troubleshoot this kind of issue?

Thanks
Steve

**Comment by tomfado** [comment]
Hi Steve

To get an idea of what is happening, use Terminal and run this command:

qlmanage -d 1 -p <your document>

Regards
Tomas Fasth

Steve Ottoz skrev den 2015-07-28 23:17:
> Hi
> 
> I installed the plugin a while ago and everything was working fine, but
> now all of a sudden it isn't.
> I didn't really change anything, at least not that I know of. And there
> was no big OS update or anything.
> There also isn't any other QL plugin interfering, because if I remove
> all the other plugins and only keep QLStephen, it's still not working.
> 
> Is there anything you can think of, that could cause this issue? Or is
> there any way I can debug/troubleshoot this kind of issue?
> 
> Thanks
> Steve
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/whomwah/qlstephen/issues/42>.
> 

-- 
Tomas Fasth <tomfa@debian.org>
Debian Developer
GPG fingerprint: B9BC EDE2 9580 516A EA8C 34E7 33C1 FE3D 81D7 2FF8

**Comment by iamso** [comment]
Hi Tomas

That really helped. Somehow another broken plugin prevented QLStephen from executing correctly.

I recently installed the Lytro Desktop Software to test something for work. It turns out that it has it's own QL plugin, that isn't installed inside the QL plugin folder, that's why removing all installed QL plugins didn't help. So I uninstalled the app, ran `qlmanage -r` and QLStephen is now working again.

Thanks a lot.
Steve
